.github/workflows/rust.yml

```diff
@@ -69,3 +69,15 @@ jobs:
           toolchain: ${{ matrix.msrv }}
       - name: cargo +${{ matrix.msrv }} check
         run: cargo check --all-features
+  lockfile:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout repository
+      uses: actions/checkout@v4
+    - name: Install Rust
+      uses: dtolnay/rust-toolchain@stable
+      with:
+        toolchain: stable
+    - uses: Swatinem/rust-cache@v2
+    - name: "Is lockfile updated?"
+      run: cargo update --workspace --locked
```

Comment:
I'd never seen this trick before. I'll copy that for other projects :raised_hands:

Reply:
If you are curious, the settings for all of my projects is at https://github.com/epage/_rust

I have that as a merge-base for easy updating

Reply:
Thanks, that's awesome :eyes: